documentation/docs/pipelines/abapEnvironment/configuration.md

```diff
@@ -0,0 +1,132 @@
+# Configuration
+
+In this section, you can learn how to create a configuration in a (GitHub) repository to run an ABAP Environment Pipeline.
+
+## 1. Prerequisites
+
+* Configure your Jenkins Server according to the [documentation](https://sap.github.io/jenkins-library/guidedtour/).
+* Create a git repository on a host reachable by the Jenkinsserver (e.g. GitHub.com). The pipeline will be configured in this repository.
+
+## 2. Jenkinsfile
+
+Create a file named `Jenkinsfile` in your repository with the following content:
+
+```
+@Library('piper-lib-os') _
+
+abapEnvironmentPipeline script: this
+```
+
+The annotation `@Library('piper-lib-os')` is a reference to the Jenkins Configuration, where you configured the Piper Library as a "Global Pipeline Library". If you want to **avoid breaking changes** we advise you to use a specific release of the Piper Library instead of the default master branch (see [documentation](https://sap.github.io/jenkins-library/customjenkins/#shared-library)).
+
+## 3. Manifest for Service Creation
+
+Create a file `manifest.yml`. The pipeline will create a SAP Cloud Platform ABAP Environment System in the beginning (and delete it in the end). This file describes the ABAP instance, which will be created:
+
+```yaml
+---
+create-services:
+- name:   "abapEnvironmentPipeline"
+  broker: "abap"
+  plan:   "16_abap_64_db"
+  parameters: "{ \"admin_email\" : \"user@example.com\", \"description\" : \"System for ABAP Pipeline\" }"
+```
+
+The example values are a suggestion. Please change them accordingly and don't forget to enter your own email address. Please be aware that creating a SAP Cloud ABAP Environment instance may incur costs.
+
+Please have a look at the [step documentation](https://sap.github.io/jenkins-library/steps/cloudFoundryCreateService/) for more details.
+
+## 4. Configuration for the Communication
+
+The communication to the ABAP system is done using a Communication Arrangement. The Communication Arrangement is created during the pipeline via the command `cf create-service-key`. The configuration for the command needs to be stored in a JSON file. Create the file `sap_com_0510.json` in the repository with the following content:
+
+```json
+{
+  "scenario_id": "SAP_COM_0510",
+  "type": "basic"
+}
+```
+
+Please have a look at the [step documentation](https://sap.github.io/jenkins-library/steps/cloudFoundryCreateServiceKey/) for more details.
+
+## 5. Configuration for ATC
+
+Create a file `atcConfig.yml` to store the configuration for the ATC run. In this file, you can specify which Packages or Software Components shall be checked. Please have a look at the step documentation for more details. Here is an example of the configuration:
+
+```yml
+atcobjects:
+  softwarecomponent:
+    - name: "/DMO/REPO"
+```
+
+Please have a look at the [step documentation](https://sap.github.io/jenkins-library/steps/abapEnvironmentRunATCCheck/) for more details.
+
+## 6. Technical Pipeline Configuration
```

Comment:
Maybe also point out here which sections can be left out, e.g. Prepare System if the system has already been provisioned correctly as mentioned above. Although you have already declared it in below explanation maybe you could get a little more into detail or give more information what can be removed.